client/components/thank-you/theme.ts

```diff
@@ -0,0 +1,37 @@
+export type ThankYouThemeProps = { theme?: ThankYouThemeType };
+export type ThankYouThemeType = {
```

Comment:
Could you add a blank line above 🙇‍♂️? We usually try to avoid making blocks of code that are too compact as they are generally harder to read. Thanks!